template/.prettierignore

```diff
@@ -0,0 +1,7 @@
+# Ignore artifacts:
+.github
+.husky
+**/CHANGELOG.md
+api.md
+coverage
+node_modules
```

Comment:
There should be another return here.

https://prettier.io/docs/en/options.html#end-of-line